docs/installation/advanced/integrating-from-source-vite.md

```diff
@@ -294,30 +294,30 @@ You can also translate the above code using TypeScript.
 ```ts
 // main.ts
 
-import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
-import EssentialsPlugin from '@ckeditor/ckeditor5-essentials/src/essentials';
-import AutoformatPlugin from '@ckeditor/ckeditor5-autoformat/src/autoformat';
-import BoldPlugin from '@ckeditor/ckeditor5-basic-styles/src/bold';
-import ItalicPlugin from '@ckeditor/ckeditor5-basic-styles/src/italic';
-import BlockQuotePlugin from '@ckeditor/ckeditor5-block-quote/src/blockquote';
-import HeadingPlugin from '@ckeditor/ckeditor5-heading/src/heading';
-import LinkPlugin from '@ckeditor/ckeditor5-link/src/link';
-import ListPlugin from '@ckeditor/ckeditor5-list/src/list';
-import ParagraphPlugin from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
+import { Essentials } from '@ckeditor/ckeditor5-essentials';
+import { Autoformat } from '@ckeditor/ckeditor5-autoformat';
+import { Bold } from '@ckeditor/ckeditor5-basic-styles';
+import { Italic } from '@ckeditor/ckeditor5-basic-styles';
+import { BlockQuote } from '@ckeditor/ckeditor5-block-quote';
+import { Heading } from '@ckeditor/ckeditor5-heading';
+import { Link } from '@ckeditor/ckeditor5-link';
+import { List } from '@ckeditor/ckeditor5-list';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
 
 ClassicEditor
     .create( document.querySelector<HTMLElement>( '#app')!, {
```

Comment:
Use the `as ...` notation to avoid the `!`? I know we had a discussion about this but I'd like to repeat it: https://github.com/ckeditor/ckeditor5/issues/13740#issuecomment-1504184685

Reply:
I would go even further, and a simple check that shows we care about type guarding :)

```ts
const editorElement = document.querySelector( "#app" );

if ( editorElement === null || !(editorElement instanceof HTMLElement ) ) {
  throw new Error( "No editor area!" );
}
```
Or more simple one
```ts
const editorElement = document.getElementById( "app" );

if ( editorElement === null ) {
  throw new Error( "No editor area!" );
}
```